app/helpers/application_helper.rb

```diff
@@ -80,4 +80,8 @@ def css_overrides
     styles += "</style>"
     styles.html_safe
   end
+
+  def get_path(screenshot)
+    screenshot.format == "pdf" ? "#{screenshot.public_id}.png" : screenshot.path
```

Comment:
Yeah but it can also be .jpg
you can check this link for more details if what I did is not clear enough: http://cloudinary.com/blog/uploading_converting_and_generating_thumbnails_for_pdf_documents